src/lib/chat/matrix-client.ts

```diff
@@ -613,6 +613,44 @@ export class MatrixClient implements IChatClient {
     await this.matrix.redactEvent(roomId, messageId);
   }
 
+  async getRoomIdForAlias(alias: string): Promise<string | undefined> {
+    await this.waitForConnection();
+    return await this.matrix
+      .getRoomIdForAlias(alias) // { room_id, servers[] }
+      .catch((err) => {
```

Comment:
Is there a particular reason you didn't use try/catch here instead of using the promise functions?

Reply:
weirdly, `try catch` wasn't catching the error here. So i had to use `.catch()` & `.then()` :/